HackIllinois/ViewControllers/HIGroupViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  HIGroupViewController.swift
+//  HackIllinois
+//
+//  Created by HackIllinois Team on 1/27/21.
+//  Copyright © 2021 HackIllinois. All rights reserved.
+//  This file is part of the Hackillinois iOS App.
+//  The Hackillinois iOS App is open source software, released under the University of
+//  Illinois/NCSA Open Source License. You should have received a copy of
+//  this license in a file with the distribution.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class HIGroupViewController: HIGroupListViewController {
+
+    // MARK: - Properties
+    lazy var fetchedResultsController: NSFetchedResultsController<Profile> = {
+        let fetchRequest: NSFetchRequest<Profile> = Profile.fetchRequest()
+
+        fetchRequest.sortDescriptors = [
+            NSSortDescriptor(key: "firstName", ascending: true)
+        ]
```

Comment:
Add a TODO about sorting by different metrics later, or randomize them? I'm not super sure how to approach it but maybe we could do alphabetical just for now